superset/db_engine_specs/trino.py

```diff
@@ -47,6 +49,29 @@ class TrinoEngineSpec(PrestoBaseEngineSpec):
     engine = "trino"
     engine_name = "Trino"
 
+    @classmethod
+    def convert_dttm(
```

Comment:
Given that Presto and Trino have diverged in therms of their `TIMESTAMP` functions I wonder if the `PrestoBaseEngineSpec.convert_dttm` method should be moved to the `PrestoEngineSpec` class.